tests/test_checkout.py

```diff
@@ -402,3 +406,110 @@ def test(self):
                     )
                 ),
             )
+
+
+class TestCheckoutShouldHaveSelfClearingProgressBar(TestDvc):
+    clearline_pattern = "\r\x1b[K"
```

Comment:
No need to copy this over. Just extract the one from progress.py into a constant and use it here and there.

Reply:
Sorry, leftover after initial regex matching